### ISSUE TITLE
Generating explanations

### ISSUE THREAD

**Issue by shanestorks** [SawanKumar28/nile#6]
It seems that after fine-tuning the label-specific generators, actually generating all of the data takes a long time. Based on the time taken so far, it seems it would take ~30 hours per label/data partition pair. Is this expected? Are the pre-generated explanations from the pre-trained generators already available somewhere?

Thanks!

**Comment by SawanKumar28** [comment]
The generated explanations for the dev and test sets are present at [generated_explanations](https://github.com/SawanKumar28/nile/tree/master/generated_explanations)

**Comment by shanestorks** [comment]
Thanks for the pointer. Closing.
